flow/project.py

```diff
@@ -3519,8 +3469,21 @@ def _get_aggregate_from_id(self, id):
                 return aggregate_store[id]
             except KeyError:
                 pass
-        # Raise error as didn't find the id in any of the stored objects
-        raise LookupError(f"Did not find aggregate with id {repr(id)}.")
+        else:
```

Comment:
I am not super familiar with this flow, can you please redirect me to someplace where this is mentioned?
```
if condition:
    # do something
# code to compute something
else:
    # do something if condition is False
```

Reply:
`for:` / `else:` is a syntax allowed by Python that is somewhat uncommon in other languages. Here's an article explaining what it does. https://book.pythontips.com/en/latest/for_-_else.html#for-else

In this specific case, since we don't have any `break` statements in the body of the `for` loop, I can remove this `else:` block and remove one level of indention (the `else:` doesn't do anything special here). That would be cleaner, so I'll make that change.